internal/extension/registry/extension_manager.go

```diff
@@ -32,16 +35,16 @@ import (
 	"github.com/envoyproxy/gateway/proto/extension"
 )
 
-const grpcServiceConfig = `{
+const grpcServiceConfigTemplate = `{
 "methodConfig": [{
 	"name": [{"service": "envoygateway.extension.EnvoyGatewayExtension"}],
 	"waitForReady": true,
 	"retryPolicy": {
-		"MaxAttempts": 4,
```

Comment:
curious about the history of these defaults

Reply:
seems inspired by the grpc docs for retries in golang: https://github.com/grpc/grpc-go/tree/master/examples/features/retry#define-your-retry-policy

Reply:
EG default for envoy retries are:
- 2 retries
- BaseInterval: 25 ms
- MaxInterval: 250ms

For Extension Server:
- 4 retries
- Base: 100ms
- Max: 1s

I think that it's ok for the control plane defaults to be more relaxed and allow more time for extension manager to recover, compared to dataplane traffic.

Reply:
thanks for checking, lets stick to the existing one then